### PR TITLE
Add UML for AddRate, Sort Commands

### PR DESCRIPTION
**Pull request by ytingtan**

### CODE REVIEW

File: docs/diagrams/AddCommandRateSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant "a:AddCommand" as AddCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+[-> LogicManager : execute(undo)
```
Comment (pratham31012002) [inline docs/diagrams/AddCommandRateSequenceDiagram.puml:13]: Will this be `execute(undo)`.

File: docs/diagrams/AddRateSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant "ar:AddRateCommand" as AddRateCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+[-> LogicManager : execute(undo)
```
Comment (pratham31012002) [inline docs/diagrams/AddRateSequenceDiagram.puml:13]: Will this be `execute(undo)`.

File: docs/diagrams/SortSequenceDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant "s:SortCommand" as SortCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+[-> LogicManager : execute(undo)
```
Comment (pratham31012002) [inline docs/diagrams/SortSequenceDiagram.puml:13]: Will this be `execute(undo)`?

File: docs/diagrams/AddRateSequenceDiagram.puml (hunk 2 above)
Comment (ytingtan) [reply docs/diagrams/AddRateSequenceDiagram.puml]: changed thanks!

**Review by Josephaedan — APPROVED** [review]
LGTM

**Pull request merged**